ci/provisioning/post_provision_config_nodes_UBUNTU_20_04.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+post_provision_config_nodes() {
```

Comment:
(lint) The mentioned parser error was in this function. [SC1009]
(lint) Couldn't parse this brace group. [SC1073]